geometry_msgs/Pose goal
---
bool successed
string text
